views/default/elgg.js.php

```diff
@@ -65,8 +65,17 @@
 elgg.security.interval = <?php echo (int)_elgg_services()->actions->getActionTokenTimeout() * 333; ?>;
 elgg.config.language = '<?php echo (empty($CONFIG->language) ? 'en' : $CONFIG->language); ?>';
 
-// We require jquery-ui because loading the file in an AMD environment doesn't automatically
-// apply its functionality to jQuery.
+// jQuery and UI must be loaded sync in 2.x but modules should depend on these AMD modules
+define('jquery', function () {
```

Comment:
In this PR, jQuery and UI just set up standard browser globals (which we still need) and then we continue on with Require and elgg.js. 3.x plugins still won't be able to use jQuery or UI before the HTML footer, so I think this is still successful in pushing devs towards AMD.